compiler/optimizations/loopInvariantCodeMotion.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2017 Advanced Micro Devices, Inc.
```

Comment:
I also don't care about copyrights/licensing/whatever, so I'll leave that for Tmac :)